Symfony/CS/Console/Output/ProcessOutput.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Console\Output;
```

Comment:
I'm working on a change to put the output-classes (xml, json, etc.) into this same namespace. I was planning to make a PR out of that after this one is done. Should I move this class one up into the `Console`-namespace for now and propose to move it with the next PR?